### ISSUE TITLE
Adding duplicate template while nominating for deletion

### ISSUE THREAD

**Issue by neslihanturan** [commons-app/apps-android-commons#1900]
**Summary:** 

I just got a message to my talk page as: "Do you know the template {{duplicate}}? --JuTa 18:39, 13 September 2018 (UTC) " Since I nominate my duplicates for deletion with explanation duplicate. 

How about adding some predefined reasons while nominating for deletion, and adding related templates to pictures?

**Comment by whym** [comment]
Good idea. {{duplicate}} expects a file that should be kept (as 'original' to the duplicate that should be deleted), so I think it will require additional UI and method to detect (near-)duplicates from the user's uploads.

**Comment by Rishavgupta12345** [comment]
@nicolas-raoul we have better feature now in our app and so we can close this issue

**Comment by nicolas-raoul** [comment]
@Rishavgupta12345 We don't have this feature yet.
